extensions/extended-blocks/paid-blocks/README.md

```diff
@@ -0,0 +1,20 @@
+Paid Blocks
+===========
+
+This shared extension takes over providing additional behavior to a block it's considered paid.
+
+This condition can depend on a few factors: is it a simple or atomic site? have the have site a paid or free plan? However, from the perspective of paid-blocks implementation, it relies on the business-layer system.
+
+In the end, the feature will define the availability of the blocks, getting data from the Jetpack global state on the client-side `Jetpack_Editor_Initial_State`.
+
+In short and as a generic rule, if you need to set a block as the paid, you should do it in the proper place not here.
+
+### Setting blocks as paid in Jetpack
+
+You can take a look at the Extension doc, [Paid Blocks](extensions/README.md#paid-blocks) section to know more about this process.
+
+### Setting blocks as paid not registered by Jetpack
+
+Sometimes some blocks need to be defined as paid because some of their features depend on the site plan, for instance, `core/video`. It's a core block.
```

Comment:
```suggestion
Sometimes blocks which are not themselves registered by Jetpack need to be marked as paid because some of their features depend on a site plan. A good example of this is the `core/video` block which is provided by Gutenberg Core and requires a paid plan in order to enable the uploading of video files.
```